packages/cli/test/fixtures/unit/vercel-json-overrides-error/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "vercel-json-overrides-error"
+}
```

Comment:
Why is this one an error? I think you can probably reuse the same test fixture for the successful test and error test.